index.md

```diff
@@ -1,51 +1,68 @@
+## Key technologies
 
-## Core Architecture
+- [**Event Streams**](glossary.md#event-streams): This is an IBM Cloud-based [Apache Kafka](https://kafka.apache.org/) managed service, the technology used for producing and consuming the data streams.
+- [**IBM Cloud Functions**](glossary.md#ibm-cloud-functions): Using these, the Health Record Ingestion service exposes a serverless RESTful Management API that is used to control and configure the system. 
+- [**Elasticsearch**](glossary.md#elasticSearch): This is the distributed NoSQL data store that is used to store information about batches.
+
+## IBM Cloud dependencies
+
+The Health Record Ingestion service was developed on the IBM Cloud and currently does not support running on other public or private clouds. However, as part of Project Alvearie, the Health Record Ingestion service is being transitioned into an open source project, with a goal of supporting public and private cloud deployments. The latest version has been pushed to GitHub. The team continues to work on moving all development activities into the open, and making the Health Record Ingestion service support other clouds. For more information about future plans, see the [Roadmap](roadmap.md).   
+
+## Core architecture
+
+<br>
+**Figure: Components of the Health Record Ingestion service**
+<br><br>
 ![core-architecture](assets/img/architecture-core.png)
+<br><br>
 
 ### Topics
-Health data, which may include [PHI](glossary.md#phi), is written to and read from the Kafka (IBM Event Streams) topics. There **_must be_** _separate topics for each tenant and Data Integrator_ in order to meet data separability requirements. 
 
-Additional topics may be created as desired. But, in general, Kafka performs better with a small number of large topics. For each (upstream) input topic, there is an associated (downstream) notification topic for [batch status notifications](https://github.com/Alvearie/hri-api-spec/tree/master/notifications/batchNotification.json). 
+Kafka (IBM Event Streams) stores streams of records in [topics](glossary.md#topic). Health data can include [Protected Health Information (PHI)](glossary.md#protected-health-information-phi). To meet data separability requirements, **there must be separate topics for each tenant and Data Integrator**. 
+
+As needed, create additional topics. Generally, Kafka performs better with a small number of large topics. For each (upstream) input topic, there is an associated (downstream) notification topic for batch status notifications.  
+
+[API specification on GitHub](https://github.com/Alvearie/hri-api-spec/tree/master/notifications/batchNotification.json)
```

Comment:
This link here doesn't really fit and need to be with the prior paragraph. Is there a problem with the original working and link?
>For each (upstream) input topic, there is an associated (downstream) notification topic for [batch status notifications](https://github.com/Alvearie/hri-api-spec/tree/master/notifications/batchNotification.json).